qulice-checkstyle/src/test/resources/com/qulice/checkstyle/DefaultMethods.java

```diff
@@ -0,0 +1,31 @@
+/**
+ * Hello.
+ */
+package foo;
+
+import java.util.List;
+import java.util.function.Consumer;
+
+/**
+ * Simple.
+ * @version $Id $
+ * @author John Smith (john@example.com)
```

Comment:
@pinaf why not? :) This is just a text file that will be checked by checkstyle, not a production code.